contentcuration/contentcuration/tests/test_user.py

```diff
@@ -38,3 +44,25 @@ def test_content_defaults_is_dict(self):
 
         mrtest2 = User.objects.get(email="mrtest@testy.com")
         assert mrtest2.content_defaults == DEFAULT_CONTENT_DEFAULTS
+
+
+class UserInvitationTestCase(BaseAPITestCase):
+    def test_user_invitation_dedupe(self):
```

Comment:
Can you write a short docstring on what you're testing in here? Seems to be a lot.